autoscheduler/scraper/management/commands/utils/scraper_utils.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+def get_all_terms() -> List[str]:
+    """ Generates all of the terms, from 2019 until now """
+
+    # Figure out the best way to do this
+    # Might want to read in the current year?
+    years = [2013 + i for i in range(7)]
+    semesters = [1, 2, 3]
+    locations = [1, 2, 3]

```

Comment:
```suggestion
    current_year = datetime.now().year
    years = range(2013, current_year + 1)
    semesters = range(1, 4)
    locations = range(1, 4)
```
Can read in the current year like this and change these iterables to `range`s